packages/pixeloven/cli-addon-webpack/src/commands/webpack.ts

```diff
@@ -23,6 +23,9 @@ export default {
         switch (task) {
             case "build":
             case "start": {
+                /**
+                 * @todo Need to list all options in some sorta help style menu
+                 */
```

Comment:
@jimmynotjim I large number of our options were poping up as invalid so i added them to the type definition below. We should eventually think about creating a "help" option and printing that here to improve the experience. However, I think that can come later when we go in and document v6.